script/workLoggingSystem.js

```diff
@@ -135,8 +139,9 @@ $("#submitNewTask").click(function() {
     console.log(estimatedTime);
     console.log(status);
     console.log(category);
+    console.log(loggingDate);
```

Comment:
better to call creationDate rather than loggingDate

Reply:
yeah, i will change the variable name